.devcontainer/ciphereditor.code-workspace

```diff
@@ -0,0 +1,33 @@
+{
+	"folders": [
+		{
+			"name": "✨ @ciphereditor/root",
```

Comment:
Let's use 🏠 for the root.

Reply:
Auto generated by the monorepo workspace plugin

Reply:
Oh I see. When it's a convention, we may as well stick with it 🤷‍♂️

Reply:
Nah nah I like the house